client/components/checkout-form.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+class CheckoutForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      creditCard: null,
+      shippingAddress: ''
+    };
+    this.handleName = this.handleName.bind(this);
+    this.handleCredit = this.handleCredit.bind(this);
+    this.handleShipping = this.handleShipping.bind(this);
+  }
+
+  handleName(event) {
+    this.setState({ name: event.target.value });
+  }
+
+  handleCredit(event) {
+    this.setState({ creditCard: event.target.value });
+  }
+
+  handleShipping(event) {
+    this.setState({ shippingAddress: event.target.value });
+  }
+
+  render() {
+
+    let totalPrice = 0;
+    for (let i = 0; i < this.props.cartItems.length; i++) {
+      totalPrice += this.props.cartItems[i].price;
+    }
+    const priceString = totalPrice + '';
+    const priceArray = priceString.split('');
+    priceArray.splice(priceArray.length - 2, 0, '.');
+    priceArray.splice(0, 0, '$');
+    let result = '';
+    for (let i = 0; i < priceArray.length; i++) {
+      result += priceArray[i];
+    }
+
+    return (
+      <div className='container'>
+        <h1>My Cart</h1>
+        <p>Order Total: {result}</p>
+        <form onSubmit={() => { this.props.placeOrder(this.state); }}>
```

Comment:
Be sure to `event.preventDefault()` here to prevent your browser from reloading.